src/app/handlers/blacklist.handler.ts

```diff
@@ -22,13 +27,16 @@ export class BlacklistHandler implements IHandler {
       return;
     }
 
-    this._expressions.forEach(({regex, label}) => {
+    this._expressions.forEach(({ regex, label }) => {
       if (message.content.toLowerCase().match(regex)) {
+        message.author.send(
+          `Please do not share \`${label}\` links in the \`${message.guild.name}\` server.`
+        );
         this.container.messageService.sendBotReport(message);
         const rep = new Moderation.Report(
           message.guild,
           message.author.username,
-          `Please do not post ${label} links.`,
+          `${message.author.username} shared a ${label} link.`
```

Comment:
no need to include username in the description bcs its saved as part of report already